<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DaoConfigForm</class>
 <widget class="QDialog" name="DaoConfigForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>当前所有设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>华文中宋</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>当前所有参数设置记录</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QtMaterialRaisedButton" name="btnChose">
        <property name="font">
         <font>
          <family>华文中宋</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>导入选中</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QtMaterialRaisedButton" name="btnDelete">
        <property name="font">
         <font>
          <family>华文中宋</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>删除选中</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QtMaterialRaisedButton" name="btnSubmit">
        <property name="font">
         <font>
          <family>华文中宋</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>提交变更</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QtMaterialRaisedButton" name="btnCancel">
        <property name="font">
         <font>
          <family>华文中宋</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialRaisedButton</class>
   <extends>QPushButton</extends>
   <header location="global">qtmaterialraisedbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DaoConfigForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
